ward/testing.py

```diff
@@ -217,6 +219,12 @@ def get_result(self, outcome, exception=None):
             if outcome in (TestOutcome.PASS, TestOutcome.SKIP):
                 result = TestResult(self, outcome)
             else:
+                if isinstance(exception, AssertionError):
+                    exc_traceback = sys.exc_info()[-1]
```

Comment:
My understanding is that `sys.exc_info` only works in the context of `except` blocks?

I think you can get the traceback from the exception using `exc_traceback = exception.__traceback__`.

Reply:
Ah. Agree `get_result` being public method I shouldn't expect it to be in context of `except`. Will change this to use `exception.__traceback`

Reply:
> My understanding is that `sys.exc_info` only works in the context of `except` blocks?
> 
> I think you can get the traceback from the exception using `exc_traceback = exception.__traceback__`.

Ah. Agree get_result being public method I shouldn't expect it to be in context of except. Will change this to use exception.__traceback

Reply:
@darrenburns, Updated with `exception.__traceback__`.